unixfs/hamt/hamt_stress_test.go

```diff
@@ -189,104 +186,3 @@ func genOpSet(seed int64, keep, temp []string) []testOp {
 		}
 	}
 }
-
-// executes the given op set with a repl to allow easier debugging
-func debugExecuteOpSet(ds dag.DAGService, width int, ops []testOp) (*HamtShard, error) {
```

Comment:
This probably shouldn't be removed. It could be useful hot-plug debugging tool.

cc @whyrusleeping